src/TypeSpec.Extension/Emitter.Csharp/src/sdk-context-options.ts

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+import { CreateSdkContextOptions } from "@azure-tools/typespec-client-generator-core";
+
+export const azureSDKContextOptions: CreateSdkContextOptions = {
+    versioning: {},
```

Comment:
set the azure preview/stable filer back. (tcgc default behavior when no filter is set).